nf_core/lint.py

```diff
@@ -286,6 +291,43 @@ def check_readme(self):
                 self.failed.append((6, "Found a bioconda environment.yml file but no badge in the README"))
 
 
+    def check_version_consistency(self):
+        """ Check that the version variable value is numeric """
+        logging.debug('Checking if version is numeric')
+        versions = {}
+        # Get the version definitions
+        # First, from the TRAVIS_TAG env var
+        versions['travis_tag'] = os.environ.get('TRAVIS_TAG')
+        if not versions['travis_tag']:
+            self.failed.append((7, "Could not determine the release tag, TRAVIS_TAG env var was not set."))
+            return
+        
+        # Second, get version from nextflow.config
+        if not 'params.version' in self.config.keys():
```

Comment:
This should never fail, as an earlier test requires it and we won't get this far if it failed.

Reply:
old habit to check key presence